cmd/context/utils.go

```diff
@@ -369,3 +369,21 @@ func fileExistsAndNotDir(filename string) bool {
 	}
 	return !info.IsDir()
 }
+
+func GetOktetoManifestPath(file string) string {
```

Comment:
@pchico83 not sure this func should be at this package, thought moving it to utils?

Reply:
@ifbyol do we already have a function for this?

Reply:
We have a similar function here: https://github.com/okteto/okteto/blob/7bc650c344105b0a76002a5eb81dc0ad8ae0b6e7/cmd/context/utils.go#L339.

It is not exactly the same but I think we should combine them to avoid to have several functions to do the same. Ideally, we should have just 1 function to load an `okteto.yaml` manifest